__tests__/app.test.js

```diff
@@ -164,3 +164,64 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("201: responds with the posted comment object with the correct properties", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({ username: "dav3rid", body: "this game sucks" })
+      .expect(201)
+      .then((response) => {
```

Comment:
Make sure your comment response is coming back as an object and not as an array of one object - you'll have to check this in your code too! We're only ever expecting one thing back in the response as we're only ever posting one thing - so make sure it's an object.